fh-messaging/lib/fhsrv.js

```diff
@@ -95,6 +98,80 @@ function handleHeadResponse(err, exists, res) {
   return;
 }
 
+function handleTopicPost(req, res) {
+  connections.num += 1;
+  connections.last = Date.now();
+  var timings_start = Date.now();
+  var numMessages = 1;
+  var topic = req.params.topic;
+  var messages = req.body;
+  var url = req.url;
+  if (messages instanceof Array) {
+    numMessages = messages.length;
+  }
+  logger.info("POST: " + url + " - topic: " + topic + ' - ' + numMessages + ' messages');
+
+  // any auth (millicore api or )
+  if (!(helpers.isValidAuth(req) || topicExcludedFromAuth(topic))) {
```

Comment:
It might not be, [here](https://github.com/feedhenry/fh-messaging/blob/master/fh-messaging/lib/fhmsg.js#L111) you can see that the `deadLetter` function creates a collection for the topic and stores the messages in mongo so this could possibly flood mongo too